src/asInput/index.jsx

```diff
@@ -5,7 +5,7 @@ import classNames from 'classnames';
 
 import newId from '../utils/newId';
 import styles from './asInput.scss';
-import ValidationMessage from '../ValidationMessage/index';
+import InvalidMessage from '../InvalidMessage/index';
```

Comment:
Nit: I know you're just replacing the `ValidationMessage` with `InvalidMessage` but I don't think the `/index` is necessary?

Reply:
Makes sense 👍